www/templates/html/MediaPlayer.tpl.php

```diff
@@ -4,15 +4,175 @@
 } else {
     echo $savvy->render($context->media, 'MediaPlayer/Audio.tpl.php');
 }
+
+$jsonTrack = @file_get_contents($context->media->getVideoTextTrackURL("json")); 
+
 ?>
 
+<?php if($jsonTrack): ?>
```

Comment:
This might be best to output in the audio/video templates separately and _only_ if there are caption tracks (logic already exists in those templates).